pkg/awsutils/mocks/awsutils_mocks.go

```diff
@@ -152,34 +152,34 @@ func (mr *MockAPIsMockRecorder) GetAttachedENIs() *gomock.Call {
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetAttachedENIs", reflect.TypeOf((*MockAPIs)(nil).GetAttachedENIs))
 }
 
-// GetENILimit mocks base method
-func (m *MockAPIs) GetENILimit() (int, error) {
+// GetENIIPv4Limit mocks base method
+func (m *MockAPIs) GetENIIPv4Limit() (int, error) {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "GetENILimit")
+	ret := m.ctrl.Call(m, "GetENIIPv4Limit")
 	ret0, _ := ret[0].(int)
 	ret1, _ := ret[1].(error)
 	return ret0, ret1
 }
 
-// GetENILimit indicates an expected call of GetENILimit
-func (mr *MockAPIsMockRecorder) GetENILimit() *gomock.Call {
+// GetENIIPv4Limit indicates an expected call of GetENIIPv4Limit
+func (mr *MockAPIsMockRecorder) GetENIIPv4Limit() *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetENILimit", reflect.TypeOf((*MockAPIs)(nil).GetENILimit))
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetENIIPv4Limit", reflect.TypeOf((*MockAPIs)(nil).GetENIIPv4Limit))
 }
 
-// GetENIIPv4Limit mocks base method
-func (m *MockAPIs) GetENIIPv4Limit() (int, error) {
+// GetENILimit mocks base method
+func (m *MockAPIs) GetENILimit() (int, error) {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "GetENIIPv4Limit")
+	ret := m.ctrl.Call(m, "GetENILimit")
 	ret0, _ := ret[0].(int)
 	ret1, _ := ret[1].(error)
 	return ret0, ret1
 }
 
-// GetENIIPv4Limit indicates an expected call of GetENIIPv4Limit
-func (mr *MockAPIsMockRecorder) GetENIipLimit() *gomock.Call {
```

Comment:
This was the wrong rebase. The comment is right, but the mock method name is wrong.